src/core/lv2/Lv2Manager.cpp

```diff
@@ -58,7 +58,77 @@ const std::set<const char*, Lv2Manager::CmpStr> Lv2Manager::pluginBlacklist =
 	"http://calf.sourceforge.net/plugins/StereoTools",
 	"http://calf.sourceforge.net/plugins/TapeSimulator",
 	"http://calf.sourceforge.net/plugins/TransientDesigner",
-	"http://calf.sourceforge.net/plugins/Vinyl"
+	"http://calf.sourceforge.net/plugins/Vinyl",
+
+	// Visualization, meters, and scopes etc., won't work until we have gui support
+	"http://distrho.sf.net/plugins/ProM",
+	"http://distrho.sf.net/plugins/glBars",
+	"http://gareus.org/oss/lv2/meters#spectr30mono",
+	"http://gareus.org/oss/lv2/meters#spectr30stereo",
+	"http://gareus.org/oss/lv2/meters#bitmeter",
+	"http://gareus.org/oss/lv2/meters#BBCM6",
+	"http://gareus.org/oss/lv2/meters#BBCmono",
+	"http://gareus.org/oss/lv2/meters#BBCstereo",
+	"http://gareus.org/oss/lv2/meters#DINmono",
+	"http://gareus.org/oss/lv2/meters#DINstereo",
+	"http://gareus.org/oss/lv2/meters#EBUmono",
+	"http://gareus.org/oss/lv2/meters#EBUstereo",
+	"http://gareus.org/oss/lv2/meters#EBUr128",
+	"http://gareus.org/oss/lv2/meters#BBCM6",
+	"http://gareus.org/oss/lv2/meters#dr14mono",
+	"http://gareus.org/oss/lv2/meters#dr14stereo",
+	"http://gareus.org/oss/lv2/meters#K12mono",
+	"http://gareus.org/oss/lv2/meters#K12stereo",
+	"http://gareus.org/oss/lv2/meters#K14mono",
+	"http://gareus.org/oss/lv2/meters#K14stereo",
+	"http://gareus.org/oss/lv2/meters#K20mono",
+	"http://gareus.org/oss/lv2/meters#K20stereo",
+	"http://gareus.org/oss/lv2/meters#NORmono",
+	"http://gareus.org/oss/lv2/meters#NORstereo",
+	"http://gareus.org/oss/lv2/meters#COR",
+	"http://gareus.org/oss/lv2/meters#dBTPmono",
+	"http://gareus.org/oss/lv2/meters#dBTPstereo",
+	"http://gareus.org/oss/lv2/meters#TPnRMSmono",
+	"http://gareus.org/oss/lv2/meters#TPnRMSstereo",
+	"http://gareus.org/oss/lv2/meters#VUmono",
+	"http://gareus.org/oss/lv2/meters#VUstereo",
+	"http://gareus.org/oss/lv2/meters#goniometer",
+	"http://gareus.org/oss/lv2/meters#stereoscope",
+	"http://gareus.org/oss/lv2/meters#SigDistHist",
+	"http://gareus.org/oss/lv2/tuna#one",
+	"http://gareus.org/oss/lv2/tuna#two",
+	"http://gareus.org/oss/lv2/sisco#Mono",
+	"http://gareus.org/oss/lv2/sisco#Stereo",
+	"http://gareus.org/oss/lv2/spectra#Mono",
+	"http://lsp-plug.in/plugins/lv2/latency_meter",
+	"http://lsp-plug.in/plugins/lv2/spectrum_analyzer_x1",
+	"http://lsp-plug.in/plugins/lv2/spectrum_analyzer_x2",
+	"http://lsp-plug.in/plugins/lv2/phase_detector",
+	"http://lsp-plug.in/plugins/lv2/profiler_mono",
+	"http://lsp-plug.in/plugins/lv2/profiler_stereo",
+	"http://invadarecords.com/plugins/lv2/meter",
+	"http://guitarix.sourceforge.net/plugins/gxtuner#tuner",
+	"https://github.com/jpcima/ADLplug",
+	"https://github.com/HiFi-LoFi/KlangFalter",
+	"https://github.com/klangfreund/SpectrumAnalyser",
+	"https://github.com/klangfreund/lufsmeter",
+	"https://github.com/laixinyuan/StereoSourceSepartion",
+	"urn:juce:TalFilter2",
+	"urn:juce:Vex",
```

Comment:
With "worker", we will get ZynAddSubFX, too, so can you please add "http://zynaddsubfx.sourceforge.net" ?